Open event generation notebook example

zlmarshall opened the pull request:
### Summary

Until now, we were missing an example of usage of the event generation open data. This adds a notebook with a few examples of using those open data. Should help @Soap2G with his presentation next week (and me with mine in 3 weeks).

### List of changes proposed in this PR (pull-request)

* New open evgen notebook featuring the use of `pyhepmc` to make some simple graphs of events, and to loop through the HEPMC events and make a couple of histograms. We also install and run Delphes as a simple parametric detector simulation and play with the output of that. This has been tested on binder and is working great. On Colab graphviz doesn't do the things I want, so I'd just focus on binder for the time being for this one.

### What should a reviewer concentrate their feedback on?

Totally new notebook! 

- [x] Documentation looks ok? Explanations are clear?
- [x] Saved output seems ok and agrees with our standards?

marianaiv reviewed (commented):
Thanks, it's great to have this! I have a couple comments, but this won't let me comment on the raw file, so let me know if anything is unclear :
- Cell 2,3, and 4: I think those initial multiline comments should be markdown cells. (minor, super picky, subcomment: all these start with "now", maybe vary this?)
- Md cell before cell 5: bold "open the file and print the first event in text form"
- Cell 5: this output is too long. Can you print less lines? First five-ten or so, or maybe remove it?
- Cell 7 to 10: same thing about the multiline comments on cell 2-4, I think it's better to have markdown cells to present the code than multiline codes in the comment.

Another thing: the image cannot be seen in mybinder (it looks like a bunch of random letters and digits). Although it can be seen in GitHub, I don't know if it's a good idea to keep the output, but I am not against it because I am not sure what is best :)

zlmarshall commented:
Thanks! I've split out the comments into markdown cells and given some more context to a few of the cells. I also added some headers to break things up a little. I vote to keep the images for now. Let me know if you have any other suggestions!
